docs/atomic-sign.1.md

```diff
@@ -0,0 +1,55 @@
+% ATOMIC(1) Atomic Man Pages
+% Brent Baude
+% August 2016
+# NAME
+atomic-sign- Create a signature for a local image
+# SYNOPSIS
+**atomic sign**
+[**-h**|**--help**]
+
+[**-o**, **--output**]
+[**--sign_key**]
+[ image ... ]
+
+# DESCRIPTION
+**atomic sign** will create a local signature for one or more local images that have 
+been pulled from a registry. Unless overridden, the signature will end up in a directory
+as follows:
+
+ /var/lib/atomic/containers/registry_name/image_name@sha256:manifest/
+
+The signatures are named 'signature-INTEGER' where the integer is increased by 1 each
+time.
+
+The default storage location for signatures is /var/lib/atomic/containers.  A different
+default location can be defined in /etc/atomic.conf with the key **default-sigstore-path**.
+
+# OPTIONS
+**-h** **--help**
+  Print usage statement.
+
+**-o** **--output**
+  File path for storing the signature, overriding the default signature
+  location, /var/lib/atomic/containers.
+
+**--sign_key**
+  Override the default signer and use the defined signer instead. You can define
+  a default in /etc/atomic.conf with the key **default_signer**.
```

Comment:
“signer” is, to my knowledge, not a common way to refer to a signing private key.

(And all variables, option names and config item names should use the same terminology; `sign_key` and `default_signer` doesn’t match.)